shared/actions/config/index.js

```diff
@@ -172,7 +181,6 @@ export function bootstrap (opts?: BootstrapOptions = {}): AsyncAction {
             dispatch(navBasedOnLoginState())
             dispatch((resetSignup(): Action))
           }
-          dispatch(registerListeners())
```

Comment:
only do this once. else if you restart the daemon we get warnings about duplicate incoming handlers. In general with the global listeners we want to register them once ever